src/main/kotlin/com/cognifide/gradle/aem/instance/tail/InstanceTailer.kt

```diff
@@ -155,7 +161,13 @@ class InstanceTailer(val aem: AemExtension) {
         val logFile = logFiles.main(instance.name)
         aem.logger.lifecycle("Tailing logs to file: $logFile")
 
-        return LogTailer(source, destination, logAnalyzerChannel)
+        return LogTailer(source, destination, InstanceLoggingInfo.of(instance), logAnalyzerChannel, consolePrinter(instance))
+    }
+
+    private fun consolePrinter(instance: Instance) = if (console) {
+        ConsolePrinter(InstanceLoggingInfo.of(instance)) { aem.logger.lifecycle(it) }
```

Comment:
`Logging` ? we used `Log` in other places.


"no matter how as long as consistent" my motto

Reply:
OK